CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Change Log
 
+# [1.2.0 (June 29th, 2021)](https://github.com/eclipse/eclipse.jdt.ls/milestone/92?closed=1)
+ * enhancement - Add a preference to control the severity of not covered maven plugin execution. See [#1770](https://github.com/eclipse/eclipse.jdt.ls/pull/1770).
+ * enhancement - Allow folding `static` blocs. See [#1777](https://github.com/eclipse/eclipse.jdt.ls/issues/1777).
+ * enhancement - Add deprecated property to CompletionItem and SymbolInformation. See [#695](https://github.com/eclipse/eclipse.jdt.ls/issues/695).
+ * enhancement - deprecated is not populated in textDocument/documentSymbol. See [#1301](https://github.com/eclipse/eclipse.jdt.ls/issues/1301).
+ * bug fix - Cannot make a static reference to the non-static type T. See [#1781](https://github.com/eclipse/eclipse.jdt.ls/issues/1781).
+ * bug fix - Standard language server should also be able to exit on shutdown. See [#1808](https://github.com/eclipse/eclipse.jdt.ls/pull/1808).
```

Comment:
https://github.com/eclipse/eclipse.jdt.ls/pull/1790 is also brought by this new release.

Introduce an extended capability `shouldLanguageServerExitOnShutdown` to control whether to exit the language server on shutdown.

Reply:
Yup. I'll add this as well. Both are now controlled by the same property (when I realized the standard server has the same issue), but I'll mention the issue. above.